src/main/rx/spektrum.h

```diff
@@ -20,5 +20,5 @@
 #define SPEKTRUM_SAT_BIND_DISABLED 0
 #define SPEKTRUM_SAT_BIND_MAX 10
 
-uint8_t spektrumFrameStatus(void);
```

Comment:
The current spektrum setup is definitely lacking. At least from the binding side of things. Having to use the CLI to bind is complicated for new users, and tedious for all users. We made that a little easier to do with the autobind receiver we released, but I personally am not a fan of autobind and there are already many existing users that dont have spektrum remote receivers with autobind.
This is how I think the firmware and configurator should work to make spektrum setup easier.
  1. Choosing between 1024 or 2048 resolution shouldn't be an option. It's confusing for people that dont have a good understanding of the receiver bind types and usually requires some research for people to figure out what they should select. The options should be either DSM2 or DSMX remote.
  2. Based on above option, binding startup pulses will be either 5 for DSM2 or 9 for DSMX. With these two bind types, receivers will provide their bind type and resolution in the first byte of a packet. This detection and autoset have been implemented in the changes I've submitted in the pull request.
  3. Bind plugs should be set on all boards for CH1 input (this makes most sense since this pin has the proper spacing from the ground pin on most boards from what I've seen). It would also only work if spektrum serial rx is configured in the settings since you wouldn't be using this pin for PWM or PPM then. This would make field binding much simpler than it currently is. 
  4. In addition, the configurator should have a gui checkbox to enter bind mode on reboot. Bind pulses would also be based on whether DSM2 or DSMX is selected.

If all your radios are set to the same travel, they should be pretty close in the endpoints. I dont have a quad or cleanflight board around me to check this, but I've never had to set specific travel % per transmitter, I use the same amount on all of them. 
Yes the change will offset the current endpoints people have set up, but its not going to make the quads unflyable either. Its a simple adjustment.
If endpoints are configurable or can be calibrated on cleanflight, then I dont see a reason why people can't still keep 147 as their travel after the changes to maintain resolution. There are people though that like to use a lower than the normal min value to trigger failsafe, and having near limit travel is going to take up the entire data range to be able to do that.
I think the changes needed to make things easier would be for the most part on the configurator side, but as I mentioned above I still think there are some things that have to be changed in the firmware.
For the data scaling portion, I can remove that from this pull request until a configurator change is done that could make the change less impacting for existing users. Ultimately though, we'd like this scaling to be implemented to maintain consistency across products and their different RC output protocols. Does that seem like an appropriate approach?